lib/rust/ensogl/component/text/src/component/selection.rs

```diff
@@ -1,5 +1,7 @@
 //! Provides a text cursor and selection component.
 
+#![cfg_attr(not(target_arch = "wasm32"), allow(dead_code))]
```

Comment:
I dont like such things globally. You never know what it refers to. Would it be way uglier to add it where its really needed?